tg_bot.py

```diff
@@ -14,7 +14,7 @@
 
 class Telegram_Bot(object):
 
-    def __init__(self, twitter_api, dirpath,level):
+    def __init__(self, twitter_api, dirpath,level, isretweet, manual_tweet):
```

Comment:
same in this case remove the params

Reply:
i think we already removed that as of latest commit.